app/models/school.rb

```diff
@@ -53,6 +53,9 @@ class School < ApplicationRecord
     where.not(id: Partnership.joins(:cohort).where(cohorts: { start_year: year }).select(:school_id))
   }
 
+  has_one :latest_school_authority, -> { latest }, class_name: "SchoolLocalAuthority"
+  delegate :local_authority, to: :latest_school_authority, allow_nil: true
+
```

Comment:
Nice!

I think we could use this in `app/views/admin/schools` - mind sneaking it in there? :)

Reply:
@kicferk1 Actually, we could even do it better. Code updated and admin/schools fixed as well.